HTML/CSS/JS keywords Pt. 2

Dusch4593 opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->

For our SEO target keyword efforts

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

<!-- Please check ALL the boxes: -->

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

SSwiniarski commented on content/html/concepts/tags/terms/ul/ul.md:
```diff
@@ -0,0 +1,40 @@
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<ul>`** tag represents an unordred list of items.
```
```suggestion
The **`<ul>`** element represents an unordered list of items.
```

SSwiniarski commented on content/html/concepts/tags/terms/ul/ul.md:
```diff
@@ -0,0 +1,40 @@
+---
+Title: '<ul>'
+Description: ''
```
```suggestion
Description: ' Represents an unordered list of items.'
```

SSwiniarski commented on content/html/concepts/iframes/iframes.md:
```diff
@@ -14,13 +14,13 @@ CatalogContent:
 ---
 
-IFrame (Inline Frame) is an HTML document embedded inside another HTML document on a website. It provides a smaller version of a browser window that contains various media, such as an advertisement, a YouTube video, etc. Some sites such as Youtube will provide iframe markup for the user to embed instead of making one on their own.
+An **IFrame (Inline Frame)** is an HTML document embedded inside another HTML document on a website. The [`<iframe>`](https://www.codecademy.com/resources/docs/html/tags/iframe) tag provides a smaller version of a browser window that contains various media, such as an advertisement, a YouTube video, etc. Some sites such as Youtube will provide IFrame markup for the user to embed instead of making ...
```
```suggestion
An **IFrame (Inline Frame)** is an HTML document embedded inside another HTML document on a website. The [`<iframe>`](https://www.codecademy.com/resources/docs/html/tags/iframe) element provides a smaller version of a browser window that contains various media, such as an advertisement, a YouTube video, etc. Some sites such as Youtube will provide IFrame markup for the user to embed instead of making one on their own.
```

SSwiniarski reviewed (changes requested):
@Dusch4593, I did a first review. Many of the changes I did were in line with my understanding of the difference between HTML tags and elements, tags being the textual markers that define the beginning and end of an element-- so it makes more sense saying that a form **element** contains UI **elements**, than a form **tag** contains UI **tags**.

Dusch4593 commented:
@SSwiniarski I implemented your feedback (which I totally agree with 😸). But I was wondering, to buttress your point about "tags vs. elements" should I move the `terms/` folder from `html/content/tags` to html/content/elements`?

Dusch4593 commented:
I made some changes in cf89e43294970e4126eb110ca6c5949ffe5b0584 that added a _lot_ more files to this PR (this is intentional). These changes include:

- Revised the "tag/element" dichotomy in the various term entries.
- Moved all the `terms/` entries to the `html/concepts/elements/` folder.
- Fixed any term entry links that would have broke due to the the folder change.
- Other small, typographical edits in line with the [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/).

Dusch4593 commented:
For the next reviewer(s), while you could definitely take a look at all 78 files, the following are the ones with the focus in SEO keyword targets:

* `content/css/concepts/background/terms/background-color/background-color.md`
* `content/css/concepts/colors/terms/hsl/hsl.md`
* `content/html/concepts/forms/forms.md`
* `content/html/concepts/iframes/iframes.md`
* `content/html/concepts/lists/lists.md`
* `content/html/concepts/semantic-html/terms/nav/nav.md`
* `content/html/concepts/elements/terms/br/br.md`
* `content/html/concepts/elements/terms/button/button.md`
* `content/html/concepts/elements/terms/dd/dd.md`
* `content/html/concepts/elements/terms/dl/dl.md`
* `content/html/concepts/elements/terms/dt/dt.md`
* `content/html/concepts/elements/terms/form/form.md`
* `content/html/concepts/elements/terms/iframe/iframe.md`
* `content/html/concepts/elements/terms/input/input.md`
* `content/html/concepts/elements/terms/li/li.md`
* `content/html/concepts/elements/terms/menu/menu.md`
* `content/html/concepts/elements/terms/ol/ol.md`
* `content/html/concepts/elements/terms/ul/ul.md`
* `content/javascript/concepts/arrays/terms/indexOf/indexOf.md`
* `content/javascript/concepts/arrays/terms/reduce/reduce.md`
* `content/javascript/concepts/arrays/terms/splice/splice.md`
* `content/javascript/concepts/comments/comments.md`

KTom101 commented on content/html/concepts/elements/terms/input/input.md:
```diff
@@ -0,0 +1,82 @@
+
+### Attributes
+
+Some common attributes used to set properties for the `<input>` element are are shown below:
```
```suggestion
Some common attributes used to set properties for the `<input>` element are shown below:
```

KTom101 reviewed (changes requested):
@Dusch4593 Here is a second review!

Dusch4593 commented:
@KTom101 I implemented your changes and I thank you for the feedback 😄

KTom101 reviewed (approved):
@Dusch4593 Looks good 👍
